tests/queries/0_stateless/02841_group_array_sorted.sql

```diff
@@ -0,0 +1,22 @@
+SELECT groupArraySorted(5)(number) from numbers(100);
+
+SELECT groupArraySorted(number) from numbers(100);
+
+SELECT groupArraySorted(100)(number) from numbers(1000);
+
+SELECT groupArraySorted(5)(str) FROM (SELECT toString(number) as str FROM numbers(10));
+
+SELECT groupArraySorted(5)(text) FROM (select toString(number) as text from numbers(10));
+
```

Comment:
I think it would be useful to test a few more data types: Nullable, LowCardinality, Array, Tuple, Decimal, IPv*, FixedString, etc.